### ISSUE TITLE
What is a question file and how to create it?

### ISSUE THREAD

**Issue by priyanthini** [CSTR-Edinburgh/merlin#507]
I'm building a Tamil (indic language) voice using Merlin. I got a very bad output. I think it might be due to the question file. I just used the default question file. I want to know the purpose of the question file and how to create a question file for a new language.

**Comment by seblemaguer** [comment]
Hello,

the question files are there: https://github.com/CSTR-Edinburgh/merlin/tree/master/misc/questions 

You have examples of multiple languages.

For Merlin, the purpose of the question file is to create the input feature vector from the label file.

**Comment by priyanthini** [comment]
@seblemaguer Thank you so much for the reply. But I want to know how to create a question file for our own language?

**Comment by seblemaguer** [comment]
Do you have a phoneme set defined?

**Comment by priyanthini** [comment]
@seblemaguer Yes I have. Is there any reference to create a question file?

Thank you.

**Comment by seblemaguer** [comment]
Not really. The question file is created in relation to the label files you are extracting from the text. Merlin is using the convention proposed in HTS. So if you don't know how it is done, I would suggest having a look at this: https://wiki.inf.ed.ac.uk/twiki/pub/CSTR/F0parametrisation/hts_lab_format.pdf

However, to have a good starting point, you simply copy the English question file to a new question file and you replace the questions related to the phonetic information from the English to your phonemic set. They are the first questions you will see in the file.

**Comment by priyanthini** [comment]
@seblemaguer Okay I will refer those and check. Thank you so much.

**Comment by priyanthini** [comment]
@seblemaguer I'm unable to create a question file for Tamil language. What can I do to solve this problem?
